src/kaltura-player.js

```diff
@@ -878,7 +878,7 @@ class KalturaPlayer extends FakeEventTarget {
 
   _handleAutoPause(visible: boolean) {
     if (!visible) {
-      if (!this.isInPictureInPicture() && this._playbackStart && !this.paused) {
+      if (!this.isInPictureInPicture() && this._playbackStart && (!this.paused || (this._adsController && this._adsController.isAdPlaying()))) {
```

Comment:
This become a very long condition, so I would maybe consider to export to function so it will be more readable.

Reply:
something like:
```js
_shouldAutoPause() {
     const isPlayingAdOrPlayback = !this.paused || (this._adsController && this._adsController.isAdPlaying();
     return !this.isInPictureInPicture() && this._playbackStart && isPlayingAdOrPlayback;
}

_handleAutoPause(visible: boolean) {
  if (!visible) {
    if (this._shouldAutoPause()) {
    ...
    }
...
}
```

Reply:
done, PTAL